api/src/main/java/io/opencensus/trace/export/ExportComponent.java

```diff
@@ -45,14 +44,22 @@ public static ExportComponent getNoopExportComponent() {
   public abstract SpanExporter getSpanExporter();
 
   /**
-   * Returns the {@link InProcessDebuggingHandler} that can be used to get useful debugging
-   * information such as (active spans, latency based sampled spans, error based sampled spans).
+   * Returns the {@link ActiveSpans} that can be used to get useful debugging information such as
+   * (active spans, latency based sampled spans, error based sampled spans).
    *
-   * @return the {@code InProcessDebuggingHandler} or {@code null} if in-process debugging is not
-   *     supported.
+   * @return the {@code SampleStore} or {@code null} if in-process debugging is not supported.
    */
   @Nullable
-  public abstract InProcessDebuggingHandler getInProcessDebuggingHandler();
+  public abstract ActiveSpans getActiveSpans();
```

Comment:
I still think it would help to give the class a more descriptive name, such as "ActiveSpansExporter" or "ActiveSpansReader".